### ISSUE TITLE
Add ability for users to customize colors

### ISSUE THREAD

**Issue by ashyon** [TootCat/mastodon#16]
As explained in https://toot.cat/@ghost/166982, some of the colors used in our theme are hard for certain users to see. We should come up with a way to specify colors for the theme in the user settings. This will be difficult, as the SCSS code is precompiled so maybe a solution to this would be multiple themes that the user can select from? That shouldn't be too difficult.

**Comment by ashyon** [comment]
Looks like @stitchxd@gnusocial.me will work on some client-side userstyles as a temporary solution while we tackle this issue.

**Comment by ubiquill** [comment]
this is cool, but would probably take a bit of work. We would probably have to add a user setting to the db as well as a widget in settings to select a theme

**Comment by ashyon** [comment]
Yeah. I'm not exactly sure how to go about it, which is why I haven't started yet.

**Comment by wavebeem** [comment]
I know there's a big demand for having a light theme too so this'll be
handled upstream eventually I'm sure.

**Comment by wavebeem** [comment]
yeah unfortunately this is an incredibly involved task imo, especially doing it in a way that won't make further development extra tricky.

**Comment by ashyon** [comment]
I guess for now we can just use a theme with some of our own accessibility tweaks
